website/docs/components/semaphore/semaphore_usage.md

```
@@ -734,4 +742,4 @@ await semaphoreFactoryFunc(semaphoreFactory);
 
 ## Further information
 
-For further information refer to [`@daiso-tech/core/semaphore`](https://daiso-tech.github.io/daiso-core/modules/Semaphore.html) API docs.
+For further information refer to [`@daiso-tech/core/semaphore`](https://daiso-tech.github.io/daiso-core/modules/Semaphore.html) API docs.
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)